### PR TITLE
modified the image hub to docker.io/gocrane and rename the deploy/manager yaml

### PR DESCRIPTION
**Pull request by yan234280533**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,13 +16,13 @@ LDFLAGS = "-X github.com/gocrane-io/crane/pkg/version.gitVersion=$(GIT_VERSION)
...
-REGISTRY ?= ccr.ccs.tencentyun.com
-REGISTRY_NAMESPACE ?= kube-orm
+REGISTRY ?= docker.io
+REGISTRY_NAMESPACE ?= gocrane
 REGISTRY_USER_NAME?=""
 REGISTRY_PASSWORD?=""
 
 # Image URL to use all building/pushing image targets
-MANAGER_IMG ?= "${REGISTRY}/${REGISTRY_NAMESPACE}/manager:${GIT_VERSION}"
+MANAGER_IMG ?= "${REGISTRY}/${REGISTRY_NAMESPACE}/crane-manager:${GIT_VERSION}"
```
Comment (mfanjie) [inline Makefile:25]: crane-manager->craned

File: deploy/crane-namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (qmhu) [inline deploy/crane-namespace.yaml:1]: we need to support deploy manager by `kubectl apply -f deploy/crane-manager` ,it's simple to use ,so i suggest to put namespace-creation.yaml in deploy/crane-manager

File: deploy/manager/crane-manager-deployment.yaml [hunk 3]
```diff
@@ -35,8 +21,8 @@ spec:
           operator: Exists
       containers:
         - name: crane-manager
-          image: ccr.ccs.tencentyun.com/kube-orm/manager:latest
+          image: docker.io/gocrane/crane-manager:latest
           imagePullPolicy: IfNotPresent
           command:
-            - /manager
+            - /crane-manager
```
Comment (mfanjie) [inline deploy/manager/crane-manager-deployment.yaml:27]: craned

File: deploy/manifests [hunk 4]
```diff
@@ -0,0 +1 @@
+../manifests
```
Comment (qmhu) [inline deploy/manifests:1]: you can rename it to `crd` and please modify the naming in Makefile

File: Makefile [hunk 5]
```diff
@@ -112,7 +112,7 @@ all: crane-manager metric-adapter
 
 .PHONY: crane-manager
```
Comment (qmhu) [inline Makefile:113]: if decide to change crane-manager to craned, please replace them all for whole project including dirs and files

File: deploy/deploy/crane-manager/deployment.yaml [hunk 6]
```diff
@@ -35,8 +35,8 @@ spec:
           operator: Exists
       containers:
```
Comment (qmhu) [inline deploy/deploy/crane-manager/deployment.yaml:36]: I think keep the folder structor now is better,  this is duplicated `deploy/deploy/crane-manager/deployment.yaml `

File: Dockerfile [hunk 7]
```diff
@@ -5,6 +5,7 @@ FROM golang:1.17.2-alpine as builder
 
 ARG LDFLAGS
 ARG PKGNAME
+ARG SUBPATH
```
Comment (qmhu) [inline Dockerfile:8]: if we decide to change crane-manager to craned, then no need to add this ARG,.

**Pull request merged**